rubin_scheduler/scheduler/schedulers/core_scheduler.py

```diff
@@ -213,21 +203,25 @@ def _check_queue_mjd_only(self, mjd):
                 result = True
         return result
 
-    def request_observation(self, mjd=None):
+    def request_observation(self, mjd=None, whole_queue=False):
         """
         Ask the scheduler what it wants to observe next
 
         Parameters
         ----------
-        mjd : `float` (None)
+        mjd : `float`
             The Modified Julian Date.
             If None, it uses the MJD from the conditions from the
             last conditions update.
+        whole_queue : `bool`
+            Return the entire list of observations in the queue (True),
+            or just a single observation (False). Default False
 
         Returns
         -------
         observation :  observation object (ra,dec,filter,rotangle)
```

Comment:
Is this returning an ObservationArray now?

Reply:
(or list of ObservationArray)?

Reply:
List of ObservationArray.

Reply:
Can you update this docstring please? 
probably something like 
observation : `list` [`~.scheduler.utils.ObservationArray`]